sphinx/quickstart.py

```diff
@@ -53,6 +53,7 @@
     'epub': False,
     'ext_autodoc': False,
     'ext_doctest': False,
+    'ext_todo': False,
```

Comment:
Isn't that already asked with: https://github.com/sphinx-doc/sphinx/pull/1712/files#diff-639ce3a52583644f64e3dce953a93a39L1225 ?